### ISSUE TITLE
Accepting system mongod binary version

### ISSUE THREAD

**Issue by MiK546** [nodkz/mongodb-memory-server#529]
## Versions

- NodeJS: 14.17.2
- mongodb-memory-server-*: 7.3.4
- mongodb: 4.4.7
- system: Linux Mint 20.1

package: mongo-memory-server-core

## What is your question?

I have forced mongo-memory-server to use the mongod binary installed to the system by setting the `MONGOMS_SYSTEM_BINARY` environmental variable to `/usr/bin/mongod`. This works fine and the system binary seems to be used in all situations as expected. But the problem is that mongo-memory-server prints complaints about the mongo version being different multiple times to the test logs. This makes reading the logs more annoying as they are filled with the same, multi-line message. The complaints are

```
getPath: MongoMemoryServer: Possible version conflict
  SystemBinary version: "4.4.7"
  Requested version:    "4.0.25"

  Using SystemBinary!
```

This can be fixed on my machine by setting the `MONGOMS_VERSION` variable to `4.4.7`. But then when I run the tests in our CI runner which is currently a few minor versions behind it complaints about the version conflicts again. Other developers in the organisation could also be a few minor versions off from me. Is there any way to force mongo-memory-server to accept the current system mongod version without warnings?

**Comment by hasezoey** [comment]
actually, this is intended to show that the requested version does not match the actual version (to notify the developer to upgrade / downgrade) to make issues / feature problems more apparent

but no, there is currently no way to disable this without setting `VERSION` to the version the system binary uses

TL;DR: this is just a warning, which can currently only be disabled with setting `VESRION` to the one the `SYSTEM_BINARY` uses

**Comment by Rush** [comment]
Wouldn't it make more sense to allow specifying version as semver? For example "^5.0.0"

**Comment by hasezoey** [comment]
@Rush because the current check also uses the `VERSION` config option that is mainly used for downloading a version, it is would not be great to also have it as a semver.

and `VERSION` itself (for download) is not a semver because it is directly used to generate the URL, there is no index download to match versions against.
